AcmeClient.php

```diff
@@ -253,13 +266,24 @@ public function finalizeOrder(CertificateOrder $order, CertificateRequest $csr,
             throw new CertificateRequestFailedException('The order has not been validated');
         }
 
-        $response = $client->request('POST', $response['certificate'], $client->signKidPayload($response['certificate'], $this->getResourceAccount(), null), false);
-        $certificatesChain = null;
-        foreach (array_reverse(explode("\n\n", $response)) as $pem) {
-            $certificatesChain = new Certificate($pem, $certificatesChain);
+        if ($order->shouldReturnAlternateCertificateIfAvailable()) {
+          $response_headers = $this->executeAcmeCertificateRequest($response['certificate'], false, true);
+
+          // Check to see if response headers include link to alternate certificate download
+          if (isset($response_headers['Link'][1])) {
```

Comment:
alternate certificate links will be sent in the response headers when retrieving a certificate from Let's Encrypt... if one is present, we need to grab that link and retrieve the cert again using that new link

```
 -headers: array:7 [
      "Cache-Control" => array:1 [
        0 => "public, max-age=0, no-cache"
      ]
      "Content-Length" => array:1 [
        0 => "4201"
      ]
      "Content-Type" => array:1 [
        0 => "application/pem-certificate-chain"
      ]
      "Link" => array:2 [
        0 => "<http://192.168.145.129:4001/directory>;rel="index""
        1 => "<http://192.168.145.129:4001/acme/cert/ff0deb708f6ce3471bc7f1ee6754925dce5d/1>;rel="alternate""
      ]
      "Date" => array:1 [
        0 => "Thu, 17 Sep 2020 17:06:33 GMT"
      ]
    ]
```